### ISSUE TITLE
Create export page

### ISSUE THREAD

**Issue by davidskalinder** [davidskalinder/mpeds-coder#57]
This should probably be a separate page in the admin interface to handle exports of various things.  Will probably need a new template (maybe copied from the old admin pages?  Though this might be simpler than that...)  Will need a JS handler for the query sigh: coder table export can be a model for this.

**Comment by davidskalinder** [comment]
To clarify, let's make this issue just be about the container page.  I'll open new issues for the specific exports.

**Comment by davidskalinder** [comment]
Mostly empty page and controller function as well as a link from the admin index created in branch create_export_page.  I'll wait to move it into testing until there's some content in it.

**Comment by davidskalinder** [comment]
Sigh okay so actually it's now merged into branch testing and live in the testing deployment so that I can follow ZenHub's (sensible) column rules.

**Comment by davidskalinder** [comment]
This was merged into testing in 82d474ec991 but has become abandonware since I haven't managed to create any exports to put in it.  So I think I'm going to revert it out of testing until I want something to go onto the page, at which point the revert should be reverted.  As discussed [here](https://sethrobertson.github.io/GitFixUm/fixup.html#pushed_new_merge), this can get complicated... But honestly, all the changes in this branch are pretty small, so even the worst case of having to write them all again wouldn't be too bad...  (NB all the changes should be in 2131a0df9aadb and 82d474ec99182bc9.)

The moral: don't merge your features before you need them.

**Comment by davidskalinder** [comment]
> This was merged into testing in 82d474e

Okay so that's a lie obviously, since 82d474e is one of the substantive commits.  I can't find the merge commit, but it doesn't matter, since what I'm actually going to do is to revert both commits in `create_export_page`, merge it into `testing`, then revert the reverts in the `create_export_page` but *not* in `testing` so that it can be merged in later.  (And I'm going to test the whole thing on a copy of `testing` first.)  Cool story bro.

**Comment by davidskalinder** [comment]
Done.  Feature reverted at e9a620ee5893 and 396b2388af1 in the feature branch, merged into testing at da4fdef150e, then un-reverted in the feature branch at cdc7e7c07801 and 881aae3a9d4.

Now the only question in where to put this issue...

**Comment by davidskalinder** [comment]
> Now the only question in where to put this issue...

I think back to backlog is the answer.  Better than having a zombie issue in user-testing...
